code/modules/mob/mob_grab.dm

```diff
@@ -88,7 +88,7 @@
 
 /obj/item/grab/proc/progress_passive(mob/living/carbon/human/user, mob/living/victim)
 	user.grab_level = GRAB_AGGRESSIVE
-	playsound(src.loc, 'sound/weapons/thudswoosh.ogg', 25, 1, 7)
+	playsound(src.loc, 'sound/weapons/Genhit.ogg', 25, 1, 7)
```

Comment:
I don't think this one fits really.

Reply:
It's supposed to be the sound of them hitting the ground, as aggressive grab forces the floored status. But I'm open to suggestions if you have another sound in mind👍

Reply:
no prob, will leave as is